hack/source-manifests.sh

```diff
@@ -89,7 +89,7 @@ function gen_ocs_csv() {
 	pushd config/manager
 	$KUSTOMIZE edit set image ocs-dev/ocs-operator="$OCS_IMAGE"
 	popd
-	$KUSTOMIZE build config/manifests | $OPERATOR_SDK generate bundle -q --overwrite=false --version "$CSV_VERSION"
+	$KUSTOMIZE build config/manifests | $OPERATOR_SDK generate bundle -q --overwrite=false --version "$CSV_VERSION" --extra-service-accounts ocs-provider-server
```

Comment:
Let's use a variable to define extra serviceaccounts. It'll be easier to add more accounts later.
If no extra accounts are provided, this arg can be skipped.

Reply:
made changes